static/app/views/issueDetails/streamline/solutionsHubDrawer.tsx

```diff
@@ -141,20 +146,37 @@ const AiSetupDataConsent = HookOrDefault({
   defaultComponent: () => <div data-test-id="ai-setup-data-consent" />,
 });
 
+const useEnableAutofix = (groupId: string) => {
+  const api = useApi();
```

Comment:
for mutations you should persist the requests
```suggestion
  const api = useApi({persistInFlight: true});
```